src/main/java/org/folio/invoices/utils/HelperUtils.java

```diff
@@ -96,6 +99,31 @@ public static HttpClientInterface getHttpClient(Map<String, String> okapiHeaders
     return httpClient;
   }
 
+  public static CompletableFuture<InvoiceCollection> getInvoices(int limit, int offset, String query,
+      HttpClientInterface httpClient, Context ctx, Map<String, String> okapiHeaders, Logger logger, String lang) {
+    CompletableFuture<InvoiceCollection> future = new VertxCompletableFuture<>(ctx);
+    final String SEARCH_PARAMS = "?limit=%s&offset=%s%s&lang=%s";
+    String GET_INVOICE_BY_QUERY = resourcesPath(INVOICES) + SEARCH_PARAMS;
```

Comment:
sonarQube doesn't like how this is NAMED_IN_ALL_CAPS

Reply:
it's also complaining about code duplication...  You may need to refactor this method to get past the SQ check

Reply:
Yes, I'm looking into it.